src/elements/content-uploader/UploadState.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import { HatWand } from '@box/blueprint-web-assets/illustrations/Medium';
+
+import UploadEmptyState from '../../icons/states/UploadEmptyState';
+import UploadSuccessState from '../../icons/states/UploadSuccessState';
+import UploadStateContent from './UploadStateContent';
+import type { View } from '../../common/types/core';
+
+import { VIEW_ERROR, VIEW_UPLOAD_EMPTY, VIEW_UPLOAD_IN_PROGRESS, VIEW_UPLOAD_SUCCESS } from '../../constants';
+
+import messages from '../common/messages';
+
+import './UploadState.scss';
+
+export interface UploadStateProps {
+    canDrop: boolean;
+    hasItems: boolean;
+    isFolderUploadEnabled: boolean;
+    isOver: boolean;
+    isTouch: boolean;
+    onSelect: () => void;
+    view: View;
+}
+
+const UploadState = ({
+    canDrop,
+    hasItems,
+    isOver,
+    isTouch,
+    view,
+    onSelect,
+    isFolderUploadEnabled,
+}: UploadStateProps) => {
+    let icon;
+    let content;
+    switch (view) {
+        case VIEW_ERROR:
+            icon = <HatWand className="upload-error-state" height={126} width={130} />;
```

Comment:
i see the height and width comes from the old illustration but can you try what it looks like with the default? we'll likely want to move towards using default options from blueprint instead of overriding

Reply:
yes, I tried using <HatWand /> by default but the icon is 13px * 13px so that I have to override it with original height and width